src/components/common/BackButton/BackButton.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <v-btn class="back-button" icon @click="goBack">
+    <slot />
+    <v-icon :icon="mdiArrowLeft" />
+  </v-btn>
+</template>
+
+<script setup lang="ts">
+import { mdiArrowLeft } from '@mdi/js'
+
+defineProps<{
+  goBack: () => void
+}>()
```

Comment:
This is not needed. Check https://github.com/Adamant-im/adamant-im/pull/758#discussion_r2040332138